sr_ur_arm_calibration_loader/src/sr_ur_arm_calibration_loader/sr_ur_arm_calibration_loader.py

```diff
@@ -89,13 +90,16 @@ def _get_serial_from_arm(self, arm_ip):
             stdin, stdout, stderr = client.exec_command('cat /root/ur-serial')
             arm_serial_number = stdout.readline()
             client.close()
-        except (paramiko.BadHostKeyException, paramiko.AuthenticationException, paramiko.SSHException) as e:
-            ssh_exception_message = "Failed to SSH into arm - {}".format(e.message)
+        except (BadHostKeyException, AuthenticationException, SSHException) as e:
+            ssh_exception_message = " Failed to SSH into arm - {}".format(e.message)
+        except NoValidConnectionsError as e:
+            ssh_exception_message = " Failed to SSH into arm - {}".format(e.errors)
+        except Exception as e:
```

Comment:
we dont want to handle exceptions that we dont know about

Reply:
Changed to more specific 'socket.error'